courses/serializers.py

```diff
@@ -381,6 +393,16 @@ def get_certificate(self, enrollment):
         except models.ProgramCertificate.DoesNotExist:
             return None
 
+    def get_receipt(self, enrollment):
+        """
+        Resolve a receipt for this enrollment
+        """
+        return (
+            "/receipt/{}".format(enrollment.order_id)
```

Comment:
Rather than returning the URL path, shouldn't you simply return the `Order.id` here because this is supposed to be a data API, has no link with how the front-end is supposed to route the receipt page.